templates/sample-app/res/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
Does this therefore imply I have to use Eclipse? What if I want to stick to good ol' vscode?
(Something for the backlog)

Reply:
No, it doesn't.

VSCode uses Eclipse plugins thus the settings files are the same.